src/loaders/Cache.js

```diff
@@ -1,46 +1,70 @@
 /**
  * @author mrdoob / http://mrdoob.com/
+ * @author vladgaidukov / vlad@meliar.ru
  */
 
 var Cache = {
 
-	enabled: false,
```

Comment:
I think that the cache must be enabled permanently. This will eliminate the possibility of performing repeated requests for file downloads.
If you think that there must be an option to disable the cache, I will add this, but I think that the default set it enabled. What do you think ?